docs/source/datasets.rst

```diff
@@ -2,43 +2,49 @@ Datasets
 ========
 
 To perform an systematic review, ASReview LAB requires a dataset representing
-all documents or publications obtained in a systematic search. To create such
-a dataset for a systematic review, typically an online library search is
+all records (e.g., abstracs of scientific papers) obtained in a systematic search. To create such
+a dataset for a systematic review, typically an `online library search <https://asreview.nl/the-importance-of-abstracts/>` is
 performed for all studies related to a particular topic.
 
-Each entry in the dataset should hold metadata on a paper.  Mandatory metadata
-are ``title`` and ``abstract``. Other metadata such as ``id``, ``author``,
-``date``, and ``keywords`` are optional. When using ASReview in simulation
-mode, an additional binary variable to indicate labeling decisions is
-required, called ``included`` or ``label_included``.  All entries in the
-dataset need to be screened, with the goal to select all relevant papers.
-
-For testing and demonstrating ASReview, the software offers
+It is possible to use
+`your own dataset <#using-your-own-data>`__ with unlabelled records, 
```

Comment:
Dead link.

Reply:
this link was already dead before I started editing... @SagevdBrand to which section should this have referred?

Reply:
I actually don't remember, so maybe just leave it out? I can only imagine that I might have written it somewhere else to refer to this page specifically, so then it would refer to the page you're already on.. not quite useful

Reply:
The link after is dead too. Let's resolve this in this PR.

Reply:
I have changed the hyperlinks